Src/Persistence/Migrations/Logging.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace TeamProject.Persistence.Migrations
+{
+    public partial class Logging : Migration
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['partial' is gratuitous in this context.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600534)